mavros/src/plugins/command.cpp

```diff
@@ -180,6 +181,9 @@ class CommandPlugin : public plugin::PluginBase {
 			bool is_not_timeout = wait_ack_for(*ack_it);
 			lock.lock();
 
+			if (!is_not_timeout) {
+				ROS_WARN_NAMED("cmd", "CMD: Command %u -- wait ack timeout", command);
+			}
```

Comment:
Probably better put that to `wait_ack_for()`, where you have command number in passed `CommandTransaction`.
That way you support both COMMAND_LONG and COMMAND_INT.

Reply:
I can put it to `wait_ack_for()`, but actually this function is used only in `send_command_long_and_wait()`, so even if I put the changes, it will work only for `COMMAND_LONG`.
Should I do it?

Reply:
I forgot that INT don't ACK... But i still think that better to warn in timeout func.

```cpp
if (tr.ack.wait(...) == timeout) {
    WARN(...);
    return false;
}
else {
    return true;
}

Reply:
fixed

Reply:
no, I should fix error, sorry

Reply:
now it's ok